gtsfm/configs/deep_front_end.yaml

```diff
@@ -13,14 +13,18 @@ SceneOptimizer:
 
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
-    corr_metric_dist_threshold: 3 # in px
+    eval_threshold_px: 3 # in px
+    estimation_threshold_px: 4 # for homography estimator
+    min_num_inliers_acceptance: 15
+
     matcher:
       _target_: gtsfm.frontend.matcher.superglue_matcher.SuperGlueMatcher
+      use_outdoor_model: True
```

Comment:
do we need this as exposed to the user right now?

Reply:
I was testing on some indoor and outdoor data, so I think we should expose it